app/controllers/wheel_configurations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class WheelConfigurationsController < ApplicationController
+    def index
+        render json: WheelConfiguration.all
+    end
+
+    def create
+        wheel_configuration = WheelConfiguration.create(wheel_configuration_params)
+        render json: wheel_configuration
+    end
+
+
+    private
+
+    def wheel_configuration_params
+        params.require(:wheel_configuration).permit(:color_1, :color_2)
```

Comment:
Have you been testing the requests through Postman or anything else? This won't work until you update all references of `color_1` and `color_2`.

Testing things before committing it's very important.